thefuck/rules/su.py

```diff
@@ -0,0 +1,12 @@
+def match(command):
```

Comment:
`thefuck.utils.for_app` is there for the help \o/

```suggestion
from thefuck.utils import for_app


@for_app('sudo')
def match(command):
```